shared/components/data/store-connection/index.jsx

```diff
@@ -59,7 +59,7 @@ const StoreConnection = React.createClass( {
 
 	render() {
 		if ( this.isDataLoading() ) {
-			return React.createElement( this.props.loadingPlaceholder );
+			return React.createElement( this.props.loadingPlaceholder, this.state );
```

Comment:
This PR also contains an update to `StoreConnection` to pass the state of the stores (which contain the given props, which is why the change is necessary) to `LoadingPlaceholder`. cc @gziolo